src/utils/analytics/gaSend.js

```diff
@@ -0,0 +1,5 @@
+import ga from './ga'
+
+export default (props) => {
+  ga('olympiatracker.send', ...props)
```

Comment:
Do you really want to integrate GA in m-i? If so I miss the AnalyticsId in ```init.js``` and custom events names

Reply:
you're right, I'll get rid of it for now